test/didReveal.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-env mocha */
+/* global contract assert */
+
+const utils = require('./utils.js');
+const BN = require('bignumber.js');
+
+contract('PLCRVoting', (accounts) => {
+  describe('Function: didReveal', () => {
+    const [alice, bob] = accounts;
+
+    it('should return true for a poll that a voter has revealed', async () => {
+      const plcr = await utils.getPLCRInstance();
+      const options = utils.defaultOptions();
+      options.actor = alice;
+
+      // alice commits
+      const pollID = await utils.startPollAndCommitVote(options);
+      await utils.increaseTime(new BN(options.commitPeriod, 10).add(new BN('1', 10)).toNumber(10));
+
+      // alice reveals
+      await utils.as(options.actor, plcr.revealVote, pollID, options.vote, options.salt);
+
+      // didReveal(alice, pollID)
+      const actual = await plcr.didReveal.call(options.actor, pollID.toString());
+      const expected = true;
+      assert.strictEqual(actual, expected, 'should have returned true because alice DID reveal');
+    });
+
+    it('should return false for a poll that a voter has committed but NOT revealed', async () => {
+      const plcr = await utils.getPLCRInstance();
+      const options = utils.defaultOptions();
+      options.actor = alice;
+
+      // alice commits
+      const pollID = await utils.startPollAndCommitVote(options);
+
+      // didReveal(alice, pollID)
+      const actual = await plcr.didReveal.call(options.actor, pollID.toString());
+      const expected = false;
+      assert.strictEqual(actual, expected, 'should have returned false because alice committed but did NOT reveal');
+    });
+
+    it('should return false for a poll that a voter has NOT committed NOR revealed', async () => {
+      const plcr = await utils.getPLCRInstance();
+      const options = utils.defaultOptions();
+      options.actor = alice;
+
+      // alice commits
+      const pollID = await utils.startPollAndCommitVote(options);
+      await utils.increaseTime(new BN(options.commitPeriod, 10).add(new BN('1', 10)).toNumber(10));
+
+      // alice reveals
+      await utils.as(options.actor, plcr.revealVote, pollID, options.vote, options.salt);
```

Comment:
Not necessary to have multiple actors involved. Just start a poll, have Alice do nothing, and call `didReveal` for her and that poll.